pkg/install/install.go

```diff
@@ -150,10 +150,18 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 		api.InstallPhaseBootstrap: {
 			action(i.createDNS),
 			action(func(ctx context.Context) error {
-				return i.installStorage(ctx, installConfig, platformCreds, image)
+				return i.createResourceGroup(ctx, installConfig)
+			}),
+			action(func(ctx context.Context) error {
+				return i.deployStorageTemplate(ctx, installConfig)
+			}),
+			action(func(ctx context.Context) error {
```

Comment:
I tried to think of a good way to avoid wrapping these install steps in function literals in order to pass extra arguments, but each idea I thought of seemed messier.